tests/cli/test_main.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+import numpy as np
+import tifffile
+import yaml
+from typer.testing import CliRunner
+
+from careamics.cli.main import app
+from careamics.config import Configuration
+from careamics.config.support import SupportedData
+
+runner = CliRunner()
+
+
+def test_train(tmp_path: Path, minimum_configuration: dict):
+
+    # create & save config
+    config_path = tmp_path / "config.yaml"
+    config = Configuration(**minimum_configuration)
+    config.data_config.data_type = SupportedData.TIFF.value
+    with open(config_path, "w") as file:
+        yaml.dump(config.model_dump(), file, indent=2)
+
+    # training data
+    train_array = np.random.rand(32, 32)
+    # save files
+    train_file = tmp_path / "train.tiff"
+    tifffile.imwrite(train_file, train_array)
+
+    # invoke command
+    result = runner.invoke(
+        app,
+        [
+            "train",
+            str(config_path),
+            "-ts",
+            str(train_file),
+            "-wd",
+            str(tmp_path),
+        ],
+    )
+    assert result.exit_code == 0
```

Comment:
Can we check if it created a checkpoint? That would be a good way to say that it trained.

Reply:
just see if the file exists? or also try to load it

Reply:
Added check for checkpoint existence

Reply:
> just see if the file exists? or also try to load it

just checking it, I think that's enough

Reply:
(sorry totally missed the comments)